UM/Qt/Bindings/ProfilesModel.py

```diff
@@ -108,18 +117,72 @@ def exportProfile(self, name, url):
         if not profile:
             return
 
-        error = ""
+        #Parse the fileType to deduce what plugin can save the file format.
+        #fileType has the format "<description> (*.<extension>)"
+        split = fileType.rfind(" (*.") #Find where the description ends and the extension starts.
+        if split < 0: #Not found. Invalid format.
+            Logger.log("e", "Invalid file format identifier %s", fileType)
```

Comment:
Shouldn't you return after this? Now you have an invalid identifier but still try to open it.